pages/topics/client-distribution.mdx

```diff
@@ -0,0 +1,398 @@
+
+</Alert>
+
+| Value     | URL                                                       | Description              |
+| --------- | --------------------------------------------------------- | ------------------------ |
+| stable    | https://play.google.com/store/apps/details?id=com.discord | Stable application build |
+| beta ^1^  | https://play.google.com/apps/testing/com.discord          | Beta application build   |
+| alpha ^1^ | https://groups.google.com/g/discord-android-alpha-testers | Alpha application build  |
+
+^1^ See the [Help Center article](https://support.discord.com/hc/en-us/articles/360035675191-Discord-Testing-Clients) for more information on Discord testing clients.
+
+###### iOS Release Channel
+
+| Value     | URL                                                                  | Description                 |
+| --------- | -------------------------------------------------------------------- | --------------------------- |
+| stable    | https://apps.apple.com/us/app/discord-talk-chat-hang-out/id985746746 | Stable application build    |
+| beta ^1^  | https://testflight.apple.com/join/gdE4pRzI                           | Beta application build      |
+| alpha ^1^ | <private\>                                                           | Internal application builds |
+
+^1^ See the [Help Center article](https://support.discord.com/hc/en-us/articles/360035675191-Discord-Testing-Clients) for more information on Discord testing clients.
+
+## Endpoints
+
+<RouteHeader method="GET" url="/download/{release_channel}" deprecated>
+  Get Latest Application Installer
+</RouteHeader>
+
+Redirects to the latest application installer for the provided [release channel](#desktop-release-channel) and selected platform.
+
+<Alert type="warn">
+
+This endpoint is in the process of being decommissioned in favor of [Get Latest Distributed Application Installer](#get-latest-distributed-application-installer).
+
+</Alert>
+
+###### Query String Params
+
+| Name        | Type   | Description                                                                                  |
+| ----------- | ------ | -------------------------------------------------------------------------------------------- |
+| platform    | string | The [platform](#desktop-platform-type) to get the installer for                              |
+| format? ^1^ | string | The [executable format](#desktop-executable-format) to get the installer for (default `deb`) |
+
+^1^ Only applicable to the [Linux platform](#desktop-platform-type).
+
+<RouteHeader method="GET" url="/updates/{release_channel}" deprecated>
+  Get Application Updates
+</RouteHeader>
+
+Returns information about the latest application host update for the provided [release channel](#desktop-release-channel) and selected platform.
+
+<Alert type="warn">
+
+This endpoint is in the process of being decommissioned in favor of [Get Latest Distributed Application Manifest](#get-latest-distributed-application-manifest).
```

Comment:
Possibly add a note here mentioning that linux and macOS still use it

Reply:
I think "in the process of" and the note on the endpoint cover that.